ssc/cmod_csp_tower_eqns.cpp

```diff
@@ -303,7 +303,7 @@ void Tower_SolarPilot_Capital_Costs_MSPT_Equations(ssc_data_t data)
     ssc_data_set_number(data, "csp_pt_cost_storage_mwht", csp_pt_cost_storage_mwht);
 
     ssc_data_get_number(data, "p_ref", &p_ref);
-    demand_var = NULL;
+    demand_var = 0;
```

Comment:
Can you assign this one and the others to:
  std::numeric_limits<double>::quiet_NaN();
instead of 0?

Reply:
It looks like you merged develop into this branch to resolve the merge conflict, but can you still change all of the NULLs to quiet_NaNs instead of 0s?

Reply:
Whoops! Fixed.